spec/features/log_in_spec.rb

```diff
@@ -0,0 +1,20 @@
+feature "Registered users can log in" do
+
+  let(:maker) do
+    Maker.create( name: 'Alaan',
+                  username: 'Alaanzr',
+                  email: 'Alaanzr@gmail.com',
+                  password: 'password')
+  end
+
+  scenario "A user logs in successfully when supplying the correct credentials" do
+    Maker.create(name: 'Alaan', username: 'Alaanzr', email: 'Alaanzr@gmail.com', password: 'password')
```

Comment:
Metrics/LineLength: Line is too long. [102/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)